src/tableView.js

```diff
@@ -269,7 +269,7 @@ function TableView(instance) {
       let value = that.instance.getDataAtRowProp(row, prop);
 
       if (that.instance.hasHook('beforeValueRender')) {
-        value = that.instance.runHooks('beforeValueRender', value);
+        value = that.instance.runHooks('beforeValueRender', TD, row, col, prop, value, cellProperties);
```

Comment:
In `beforeValueRender` you don't need to change anything in TD element.
To do that, you have the next hook, `beforeRenderer`.
A `row`, `col` and `prop` properties are available in `cellProperties`. So, the better option is to add only the last one argument instead.
```js
value = that.instance.runHooks('beforeValueRender', value, cellProperties);
```
Effect? The same functionality as you need and it's not a breaking change.